@fufexan Added the package but I'm unsure about the missing build inputs. Can you check what's missing?

I've added a few and now it still fails with

```
CMake Error at /nix/store/4k2h9qsps52n1v56zq03yi67dwmxa81i-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:645 (message):
  The following required packages were not found:

   - wayland-protocols
   - cairo
   - pango
   - pangocairo
   - pixman-1
   - xcursor
   - libinput
   - gbm
   - re2
```

--- a/pkgs/by-name/hy/hyprland-qtutils/package.nix
+++ b/pkgs/by-name/hy/hyprland-qtutils/package.nix
@@ -1,0 +1,49 @@
+{
+  lib,
+  gcc14Stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  hyprutils,
+  pciutils,
+  qt6,
+}:
+let
+  inherit (lib.strings) makeBinPath;
+in
+gcc14Stdenv.mkDerivation (finalAttrs: {
+  pname = "hyprland-qtutils";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "hyprwm";
+    repo = "hyprland-qtutils";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-Zp2cQOo1D7WH7ml5nV7mLyto0l19gTaWnbDb6zty1Qs=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    hyprutils
+    qt6.qtbase
+    qt6.qtsvg
+    qt6.qtwayland
+  ];
+
+  preFixup = ''
+    qtWrapperArgs+=(--prefix PATH : "${makeBinPath [ pciutils ]}")
+  '';
+
+  meta = {
+    description = "Hyprland QT/qml utility apps";
+    homepage = "https://github.com/hyprwm/hyprland-qtutils";
+    license = lib.licenses.bsd3;
+    maintainers = [ lib.maintainers.fufexan ];
+    platforms = lib.platforms.linux;
+  };
+})

--- a/pkgs/by-name/hy/hyprland/info.json
+++ b/pkgs/by-name/hy/hyprland/info.json
@@ -1,7 +1,7 @@
 {
-  "branch": "v0.46.1-b",
-  "commit_hash": "254fc2bc6000075f660b4b8ed818a6af544d1d64",
-  "commit_message": "version: bump to 0.46.1",
-  "date": "2024-12-17",
-  "tag": "v0.46.1"
+  "branch": "v0.46.2-b",
+  "commit_hash": "0bd541f2fd902dbfa04c3ea2ccf679395e316887",
+  "commit_message": "version: bump to 0.46.2",
+  "date": "2024-12-19",
+  "tag": "v0.46.2"
 }

--- a/pkgs/by-name/hy/hyprland/package.nix
+++ b/pkgs/by-name/hy/hyprland/package.nix
@@ -15,6 +15,7 @@
   git,
   hyprcursor,
   hyprgraphics,
+  hyprland-qtutils,
   hyprlang,
   hyprutils,
   hyprwayland-scanner,
@@ -84,14 +85,14 @@ assert assertMsg (!hidpiXWayland)
 
 customStdenv.mkDerivation (finalAttrs: {
   pname = "hyprland" + optionalString debug "-debug";
-  version = "0.46.1";
+  version = "0.46.2";
 
   src = fetchFromGitHub {
     owner = "hyprwm";
     repo = "hyprland";
     fetchSubmodules = true;
     rev = "refs/tags/v${finalAttrs.version}";
-    hash = "sha256-0SVRQJeKsdwaTO7pMM0MwTXyVwKNQ4m1f2mvcPnZttM=";
+    hash = "sha256-dj9dpVwpyTmUyVu4jtaIU39bHgVkoZjv6cgYfWyHc9E=";
   };
 
   postPatch = ''
@@ -197,6 +198,7 @@ customStdenv.mkDerivation (finalAttrs: {
         --suffix PATH : ${
           makeBinPath [
             binutils
+            hyprland-qtutils
             pciutils
             pkgconf
           ]

